1forge/src/config.ts

```diff
@@ -1,6 +1,8 @@
 import { Requester } from '@chainlink/external-adapter'
 import { Config } from '@chainlink/types'
 
+export const NAME = '1FORGE'
```

Comment:
Moved Adapter name from `index.ts` to `config.ts` so can be accessed inside the adapter